plugins/json/src/main/java/org/apache/struts2/json/JSONValidationInterceptor.java

```diff
@@ -72,8 +74,8 @@
 
     private static final Logger LOG = LogManager.getLogger(JSONValidationInterceptor.class);
 
-    private static final String VALIDATE_ONLY_PARAM = "struts.validateOnly";
-    private static final String VALIDATE_JSON_PARAM = "struts.enableJSONValidation";
+    static final String VALIDATE_ONLY_PARAM = "struts.validateOnly";
+    static final String VALIDATE_JSON_PARAM = "struts.enableJSONValidation";
```

Comment:
oh yes, that they are private was a pain as I started this new result in an app project :laughing: